lib/id3c/cli/command/manifest.py

```diff
@@ -403,9 +403,13 @@ def _parse(*,
     metavar = "<manifest-b.ndjson>",
     type = LocalOrRemoteFile("r"))
 
+@click.option("--ignore-case",
+    metavar = "<ignore-case>",
+    is_flag = True)
+
 @format_doc(PROVENANCE_KEY = PROVENANCE_KEY)
 
-def diff(manifest_a, manifest_b):
+def diff(manifest_a, manifest_b, ignore_case):
```

Comment:
A default `ignore_case=False` could also be added here, but may not be necessary if this function is only being called via click command.

Reply:
Ah true, let me add it in here just in case.